server/events/working_dir.go

```diff
@@ -89,17 +89,41 @@ func (w *FileWorkspace) Clone(
 		}
 		revParseCmd := exec.Command("git", "rev-parse", pullHead) // #nosec
 		revParseCmd.Dir = cloneDir
-		output, err := revParseCmd.CombinedOutput()
-		if err != nil {
-			log.Warn("will re-clone repo, could not determine if was at correct commit: %s: %s: %s", strings.Join(revParseCmd.Args, " "), err, string(output))
+		outputRevParseCmd, errRevParseCmd := revParseCmd.CombinedOutput()
```

Comment:
any reason this is renamed from `err`? I think it's better kept as `err` since it's more idiomatic

Reply:
Changed to keep err, I agree.